devtools/make_package.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+#
+# Script to make a Caliopen package
+#
+
+BASEDIR="`pwd`/.."
+DISTDIR="${BASEDIR}/dist"
+
+
+function make_python_package() {
```

Comment:
I agree with that, but it sound difficult as we must put build packages in a directory outside of the packages source namespace

/src/backend/main/py.main will build a caliopen_main.wheel in /dist directory for example

And in fact build process will be the same for all packages in a given language, that why I think this make_package.sh with a -t <language> option can make things simpler than repeating bulld instruction in all packages.